docker-compose.yml

```diff
@@ -59,9 +59,18 @@ services:
   memcached:
     <<: *restart_policy
     image: "memcached:1.5-alpine"
+    healthcheck:
+      test: echo stats | nc 127.0.0.1 11211
+      interval: 10s
+      retries: 60
   redis:
     <<: *restart_policy
     image: "redis:5.0-alpine"
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
+      interval: 1s
```

Comment:
Why do you check redis way more frequently than memcached?

Reply:
Well this is because Redis has a real check and memcached uses some specific commands to preform some kind of check
That said I can research a bit more to find a better way to check memcached

Reply:
Maybe we can add a comment about this?

Reply:
> Well this is because Redis has a real check and memcached uses some specific commands to preform some kind of check

I was stupid, this was indeed the right command as it asks stats to memcached and it would not respond if it was down
it's not an official way but enough for PerconaDB to post an article about it: https://www.percona.com/blog/2008/11/26/a-quick-way-to-get-memcached-status/

Reply:
Great to hear, except for the part you call yourself stupid 😀